ibis/backends/flink/tests/test_udf.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from ibis import udf
+
+
+def test_builtin_scalar_udf(con):
+    @udf.scalar.builtin
+    def parse_url(string1: str, string2: str) -> str:
```

Comment:
I actually also wanted to add a test for the 3-argument variant, but I couldn't find a "reasonable" way to do this. It seems like both UDFs would need to have the name `parse_url`, so that would mean I'd need to construct them in a way that doesn't overwrite each other. There's no way to make an optional argument.

In the end, I didn't add this workaround in the tests, because that's not how a user would use it; would I be correct to assume a user can't have both `PARSE_URL(string1, string2)` and `PARSE_URL(string1, string2, string3)` in an Ibis expression?

Reply:
We don't have a way of overloading functions at the moment, so yeah a user would have to encode these as separate functions:

```python
@udf.scalar.builtin(name="parse_url")
def parse_url2(a, b):
	...

@udf.scalar.builtin(name="parse_url")
def parse_url3(a, b, c):
	...
```

Reply:
Ah, I didn't see the `name` argument; that works then!